vulyk/models/tasks.py

```diff
@@ -42,16 +56,21 @@ def as_dict(self):
         }
 
     def __unicode__(self):
-        return unicode(self.id)
+        return six.text_type(self.id)
 
     def __str__(self):
         return self.__unicode__()
 
     def __repr__(self):
-        return unicode(self.id)
+        return self.__unicode__()
 
 
 class AbstractAnswer(Document):
+    """
+    This is AbstractTask model.
+    You need to inherit it in your model
+    """
+    id = StringField(max_length=200, default='', primary_key=True)
```

Comment:
What for? Do we really need this `id` to be specified instead of mongo's default one?
Binary `_id` is faster than string one and the latter is needed only i there is certain reason to combine it manually.